operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -1214,36 +1216,31 @@ func (s *ClusterStatus) SetRestarting(restarting bool) {
 	s.DeprecatedUpgrading = restarting
 }
 
-// GetCurrentReplicas returns the current number of replicas that the controller wants to run.
-// It returns 1 when not initialized (as fresh clusters start from 1 replica)
-func (r *Cluster) GetCurrentReplicas() int32 {
-	if r == nil {
+func (r *Cluster) SumNodePoolReplicas() int32 {
```

Comment:
nit: This doesn't indicate if it's returning the current replicas or desired replicas. Please add a comment or rename the method.

Reply:
renamed to GetDesiredReplicas. it looks like it in the diff, but this function is not a replacement for GetCurrentReplicas - this can be seen in metric_controller.go, where a reference to Spec.Replicas is replaced with a call to this function here.